tracer/transport.go

```diff
@@ -22,6 +25,24 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation that sends traces to
+// a trace agent running on the given hostname and port. If the zero values for
+// hostname and port are provided, the default values will be used ("localhost"
+// for hostname, and "7777" for port).
+//
+// In general, using this method is only necessary if you have a trace agent
+// running on a non-default port or if it's located on another machine.
+func NewHTTPTransport(hostname, port string) Transport {
```

Comment:
Having a separated new function that allows you to return the underlying ``Transport`` mechanism is fine. We can easily combine this PR with #17 so that you can use both env vars or simply creating your tracer with this transport.

The only nitpick is to call the new function ``NewTransport`` instead of ``NewHTTPTransport`` (or something similar) because in the future we may use something different from HTTP and it would be a pain to change this exported func.